src/app/dashboard/page.tsx

```
@@ -23,37 +25,45 @@ export default async function DashboardPage() {
   switch (userProfile.role) {
     case 'JOB_SEEKER':
       return <JobSeekerDashboardView profile={userProfile} />;
+    
     case 'EMPLOYER':
-      return <EmployerDashboardView profile={userProfile} />;
+      // Fetch all required data for the employer dashboard in parallel
+      const [stats, jobPerformance, recentApplications] = await Promise.all([
+        getEmployerDashboardStatsAction(),
+        getJobPerformanceAction(),
+        getEmployerRecentApplicationsAction(),
+      ]);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)